trie/triedb/pathdb/disklayer.go

```diff
@@ -393,17 +398,24 @@ func (dl *diskLayer) revert(h *history, loader triestate.TrieLoader) (*diskLayer
 	// needs to be reverted is not yet flushed and cached in node
 	// buffer, otherwise, manipulate persistent state directly.
 	if !dl.buffer.empty() {
+		log.Info("dl buffer is not empty")
```

Comment:
For log, `dl` is meaningless and not suitable for reading. It is better to refine it.
Btw, is this log really necessary?

Reply:
Deleted